src/Conference/Admin/Meta/Speaker.php

```diff
@@ -0,0 +1,185 @@
+		] );
+
+		// Instagram URL
+		$cmb->add_field( [
+			'name'      => _x( 'Instagram URL', 'speaker meta box field', 'wp-conference-schedule' ),
+			'id'        => 'wpcsp_instagram_url',
+			'type'      => 'text_url',
+			'protocols' => [ 'http', 'https' ]
+		] );
+
+		// LinkedIn URL
+		$cmb->add_field( [
+			'name'      => _x( 'LinkedIn URL', 'speaker meta box field', 'wp-conference-schedule' ),
+			'id'        => 'wpcsp_linkedin_url',
+			'type'      => 'text_url',
+			'protocols' => [ 'http', 'https' ]
+		] );
+
+		// YouTube URL
+		$cmb->add_field( [
+			'name'      => _x( 'YouTube URL', 'speaker meta box field', 'wp-conference-schedule' ),
+			'id'        => 'wpcsp_youtube_url',
+			'type'      => 'text_url',
+			'protocols' => [ 'http', 'https' ]
+		] );
+
+		// Website URL
+		$cmb->add_field( [
+			'name'      => _x( 'Website URL', 'speaker meta box field', 'wp-conference-schedule' ),
+			'id'        => 'wpcsp_website_url',
+			'type'      => 'text_url',
+			'protocols' => [ 'http', 'https' ]
+		] );
+	}
+
+	/**
+	 * Filters session speaker meta field.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $cmb The current CMB2 box object.
+	 */
+	public function filter_session_speaker_meta_field( $cmb ) {
+		// Speaker Display Type
+		$cmb->add_field( [
+			'name'             => _x( 'Speaker Display', 'session meta field', 'wp-conference-schedule' ),
+			'id'               => 'wpcsp_session_speaker_display',
+			'type'             => 'radio',
+			'show_option_none' => false,
+			'options'          => [
+				'typed' => _x( 'Speaker Names (Typed)', 'session meta field option', 'wp-conference-schedule' ),
+				'cpt'   => _x( 'Speaker Select (from Speakers CPT)', 'session meta field option', 'wp-conference-schedule' )
+			],
+			'default'          => 'typed'
+		] );
+
+		// Fetch speakers
+		$args     = [
+			'numberposts' => - 1,
+			'post_type'   => 'wpcsp_speaker',
+		];
+		$speakers = get_posts( $args );
+		$speakers = wp_list_pluck( $speakers, 'post_title', 'ID' );
+
+		// Speaker Select Field
+		$cmb->add_field( [
+			'name'       => _x( 'Speakers', 'session meta field', 'wp-conference-schedule' ),
+			'id'         => 'wpcsp_session_speakers',
+			'desc'       => _x( 'Select speakers. Drag to reorder.', 'session meta field description', 'wp-conference-schedule' ),
+			'type'       => 'pw_multiselect',
+			'options'    => $speakers,
+			'attributes' => [
+				'data-conditional-id'    => 'wpcsp_session_speaker_display',
+				'data-conditional-value' => 'cpt'
+			]
+		] );
+
+		// Speaker Names Field
+		$cmb->add_field( [
+			'name'       => _x( 'Speaker Name(s)', 'session meta field', 'wp-conference-schedule' ),
+			'id'         => '_wpcs_session_speakers',
+			'type'       => 'text',
+			'attributes' => [
+				'data-conditional-id'    => 'wpcsp_session_speaker_display',
+				'data-conditional-value' => 'typed'
+			]
+		] );
+
+		// Fetch sponsors
+		$args     = [
+			'numberposts' => - 1,
+			'post_type'   => 'wpcsp_sponsor',
```

Comment:
constant.

Reply:
I made the updates, let me know if there is anything else. Thanks.